internal/scan-manager/scanners/qualys/pom.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add this scanner to the parent pom file (as a module). Otherwise scan-manager build will not trigger the build of this scanner. Check https://github.com/wso2/security-tools/pull/102/files#diff-4db492f31ce2da137e2b1dd5d186389a